vehicle/OVMS.V3/main/ovms_events.h

```diff
@@ -129,6 +129,7 @@ class OvmsEvents
     TimerList m_timers;
     TimerStatusMap m_timer_active;
     OvmsMutex m_timers_mutex;
+    int detect_event_loop_blockage;
```

Comment:
Does this have to be a member variable? 
Seems like it's re-initialized and only used within one function, so sound like it could be a local variable instead. 

Additional note: If it does have to be a membrer variable, it's missing the "m_" prefix.